lib/ui/semantics.dart

```diff
@@ -366,6 +380,28 @@ class SemanticsFlag {
   /// used to implement accessibility scrolling on iOS.
   static const SemanticsFlag isHidden = const SemanticsFlag._(_kIsHiddenIndex);
 
+  /// Whether the semantics node represents an image.
+  ///
+  /// Platforms have special behavior for images.  TalkBack will inform the user
+  /// the labeled node is an image. iOS may use the image flag to avoid
+  /// inverting their color when using smart invert.
+  static const SemanticsFlag isImage = const SemanticsFlag._(_kIsImageIndex);
+
+  /// Whether the semantics node is a live region.
+  /// 
+  /// A live region indicates that updates to semantics node are important. Platforms 
+  /// are free to use this information to make polite updates to the user to inform 
```

Comment:
remove "are free to", or maybe replace it with "typically" or some such. We're not giving them permission here.

Reply:
Done